### ISSUE TITLE
Add -m option for git revert

### ISSUE THREAD

**Issue by adelejjeh** [eamodio/vscode-gitlens#1639]
Currently trying to revert a merge commit causes a fatal error. Having the `-m` option integrated into gitlens' `revert` would be very helpful.

**Comment by Drarig29** [comment]
I agree. Maybe it could show a prompt to choose between the 2 parent commits?
